client/src/components/Leaderboard/Leaderboard.js

```diff
@@ -0,0 +1,101 @@
+import React from "react";
+import "./Leaderboard.scss";
+
+
+export default function Leaderboard(props) {
```

Comment:
I would not use a default export, but rather a named export:
```js
export const Leaderboard = (props) => {
```
but that remains a personal preference.